### ISSUE TITLE
Bob Briscoe's Nov 2015 review

### ISSUE THREAD

**Issue by larseggert** [dthaler/dctcp#7]
Bob's review needs to be addressed (and replied to): https://www.ietf.org/mail-archive/web/tcpm/current/msg10051.html

**Comment by pravb** [comment]
Yes I am halfway through it. I will finish it in the next couple of weeks.
